package-lock.json

```diff
@@ -3979,6 +3979,14 @@
       "integrity": "sha1-w2GT3Tzhwu7SrbfIAtu8d6gbHA8=",
       "dev": true
     },
+    "monotone-convex-hull-2d": {
```

Comment:
What's the reason for all those additions, for a deleted dependency in `package.json`?